pkg/products/amqstreams/reconciler.go

```diff
@@ -161,21 +162,21 @@ func (r *Reconciler) handleCreatingComponents(ctx context.Context, client k8scli
 		kafka.Name = clusterName
 		kafka.Namespace = r.Config.GetNamespace()
 
-		kafka.Spec.Kafka.Version = "2.1.1"
-		kafka.Spec.Kafka.Replicas = 3
+		kafka.Spec.Kafka.Version = "2.4.0"
+		kafka.Spec.Kafka.Replicas = 2
 		kafka.Spec.Kafka.Listeners = map[string]kafkav1.KafkaListener{
 			"plain": {},
 			"tls":   {},
 		}
-		kafka.Spec.Kafka.Config.OffsetsTopicReplicationFactor = "3"
-		kafka.Spec.Kafka.Config.TransactionStateLogReplicationFactor = "3"
+		kafka.Spec.Kafka.Config.OffsetsTopicReplicationFactor = "2"
+		kafka.Spec.Kafka.Config.TransactionStateLogReplicationFactor = "2"
```

Comment:
Do you know what these properties are used for?

Reply:
It wouldn't deploy with these set to the original values so I matched them based on the docs to the number of replicas in use. 

From the docs : 

- offsets.topic.replication.factor: The replication factor for the offsets topic (set higher to ensure availability). Internal topic creation will fail until the cluster size meets this replication factor requirement.

Type: short
Default: 3
Valid Values: [1,...]
Importance: high
Update Mode: read-only

- transaction.state.log.replication.factor: The replication factor for the transaction topic (set higher to ensure availability). Internal topic creation will fail until the cluster size meets this replication factor requirement.

Type: short
Default: 3
Valid Values: [1,...]
Importance: high
Update Mode: read-only

Reply:
Perfect, thanks @CathalOConnorRH